packages/material-ui/src/FormControl/FormControl.js

```diff
@@ -79,6 +101,14 @@ const FormControl = React.forwardRef(function FormControl(props, ref) {
     ...other
   } = props;
 
+  const styleProps = {
+    ...props,
+    margin,
+    fullWidth,
+  };
```

Comment:
```suggestion
  const styleProps = {
    ...props,
    color,
    component,
    disabled,
    error,
    fullWidth,
    hiddenLabel,
    margin,
    required,
    size,
    variant,
  };
```

Let's include all default props